sorc/ncep_post.fd/COLLECT_LOC.f

```diff
@@ -81,6 +84,9 @@ SUBROUTINE COLLECT_LOC ( A, B )
       deallocate(buff)
       deallocate(rbufs)
 
+       tb=mpi_wtime()
```

Comment:
@KarinaAsmar-NOAA Clean up the debugging code.

Reply:
@GeorgeVandenberghe-NOAA Would you please clean up the debugging part of COLLECT_LOC.f? Let me know when done and I'll push it to this branch.